### DISCUSSION TITLE
too many nodes in juggl view

### DISCUSSION THREAD

**Discussion by farrellyuan** [SkepticMystic/breadcrumbs#388] [Ideas]
![image](https://user-images.githubusercontent.com/104613481/165889578-76b72b31-36a9-4b23-93eb-f7d576b84ad9.png)
As it is shown by pcitures.
problem: I have too manys up direction and down direction，so every time i open breadcrumb juggl view,there is too many nodes in juggl view。
Suggestion: a new default set of maximun nodes shown in juggl view （then show more node if i want to see more node)

**Comment by farrellyuan** [comment]
![image](https://user-images.githubusercontent.com/104613481/165904779-afd3b274-4797-4682-bfd2-7516c2ae0a9a.png)
solution:i change the jscode and success in limiting maximum ring number to 3
PS：maximum ring number to 3=i only see current node's grandson'son node

**Comment by michaelpporter** [comment]
Closing — the Juggl integration was dropped in v4. Reopen if this applies to a current view.
